packages/reassure-logger/src/logger.ts

```diff
@@ -0,0 +1,64 @@
+import chalk from 'chalk';
```

Comment:
Did you remove the `hello()` and `bye()` logs? Or are they implemented in a different way? Or should it be reconciled when we merge with my PR?